README.md

```diff
@@ -43,93 +58,140 @@ interface reactDesc {
 stamp.compose(...desc?: stamp|reactDesc|specDesc[]): stamp
 ```
 
-The most powerful feature of [stamps](https://github.com/stampit-org/stamp-specification) is their composability. What this means is that `n` number of stamps can be combined into a new stamp which inherits each passed stamp's behavior. This is perfect for React, since `class` is being pushed as the new norm and does not provide an idiomatic way to use mixins. (classical inheritance :disappointed:). Stamps embrace prototypal inheritance and as a result provide great flexibility when extending React components.
+The most powerful feature of [stamps](https://github.com/stampit-org/stamp-specification) is their composability. Any number of stamps can be combined into a new stamp which inherits each passed stamp's behavior. This is perfect for React since `class` is being pushed as the new norm and does not provide an idiomatic way to use mixins. (classical inheritance :disappointed:).
+
+Let's go step by step through an example. It uses a pattern I call **Behavior Driven Composition**.
 
-__reactStamp.js__
+__container.js__
 ```js
-import React from 'react';
-import reactStamp from 'react-stamp';
+container(
+  React: object,
+  reactStamp: Function
+  components: array
+  behaviors: array
+): componentStamp
+```
+```js
+export default (
```

Comment:
> Do we really need the text behavior?
Nope. Button behavior would probably be enough.

>  I'd probably take an object literal if I were actually building a BDC for a real app...
I agree I'm not happy with the current signature. What about:

```js
container(
  React: object,
  reactStamp: object,
  ...rest: Function|object[]
): componentStamp
```